<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro</class>
 <widget class="QDialog" name="registro">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>352</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Registro de Transferencias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../banco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(187, 148, 91);</string>
   </property>
   <property name="text">
    <string>Registro de Transferencias</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Documento</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDocumento">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - Documento Nacional de Identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSS - Pasaporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VISA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Número de Documento</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDoc">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGiro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>---------Seleccione una Opción---------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios/Servicios Técnicos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Remesas/Giros</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Motivo del Giro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Monto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkTInter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Transferencia Internacional</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkDolar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Dólares</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegis">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 91, 78);
color: rgb(255, 255, 255);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
